__tests__/getEndpoints.test.js

```diff
@@ -125,6 +125,77 @@ describe("GET endpoints", () => {
           expect(articles).toBeSortedBy("created_at", { descending: true });
         });
     });
+    describe("Queries", () => {
+      test("200: articles can be filtered by existing TOPIC", () => {
+        return request(app)
+          .get("/api/articles?topic=mitch")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles.length).toBe(12);
+            body.articles.forEach((article) => {
+              expect(article.topic).toBe("mitch");
+            });
+          });
+      });
+      test("200: articles can be sorted by allowed fields", () => {
+        return request(app)
+          .get("/api/articles?sort_by=comment_count")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles).toBeSortedBy("comment_count", {
+              descending: true,
+            });
+          });
+      });
+      test("200: articles are ordered by defined order", () => {
+        return request(app)
+          .get("/api/articles?order=ASC")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles).toBeSortedBy("created_at", {
+              ascending: true,
+            });
+          });
+      });
+      test("200: accepts multiple valid queries at once", () => {
+        return request(app)
+          .get("/api/articles?topic=mitch&sort_by=comment_count&order=ASC")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles.length).toBe(12);
+            expect(body.articles).toBeSortedBy("comment_count", {
+              ascending: true,
+            });
+            body.articles.forEach((article) => {
+              expect(article.topic).toBe("mitch");
+            });
+          });
+      });
+    });
+    test("200: returns empty array when given non-existent topic", () => {
```

Comment:
This one should be a 404 if it's non-existent. 👍  We would want to do this 200 test with an empty array, but it should be for a valid topic that has no articles (i.e. "paper")